pkg/daemon/update.go

```diff
@@ -54,6 +54,9 @@ const (
 	// Crio reload will happen when /etc/containers/registries.conf is changed. This will cause
 	// a "systemctl reload crio"
 	postConfigChangeActionReloadCrio = "reload crio"
+	// GPGNoRebootPath is the path MCO expects will contain GPG key updates. MCO will attempt to only reload crio for
+	// changes to this path. Note that other files added to the parent directory will not be handled specially
+	GPGNoRebootPath = "/etc/machine-config-daemon/no-reboot/containers-gpg.pub"
```

Comment:
Is it good to make paths like this public so the user could use it? Would it be good for us to use a const from the image repo for the `policy.json` path? https://github.com/containers/image/blob/main/signature/policy_config.go#L38

Reply:
We can but I wouldn't worry about this much because implementation in OCP comes finally from container runtimeconfig controller and policy.json path is hard-coded there https://github.com/openshift/machine-config-operator/blob/master/pkg/controller/container-runtime-config/helpers.go#L38 . Also, I think OCP documentation will be referring to same path as well, so most likely it will not change unless advertised.